src/commands/deploy.ts

```diff
@@ -70,42 +60,16 @@ export abstract class DeployCommand extends Command {
 
 export default class Deploy extends DeployCommand {
   auth_required(): boolean {
-    const { flags } = this.parse(Deploy);
-    return !flags.local;
+    return true;
   }
 
-  static description = 'Create a deploy job on Architect Cloud or run stacks locally';
+  static description = 'Create a deploy job on Architect Cloud';
 
   static flags = {
     ...DeployCommand.flags,
     ...AccountUtils.flags,
     ...EnvironmentUtils.flags,
 
-    local: flags.boolean({
-      char: 'l',
-      description: 'Deploy the stack locally instead of via Architect Cloud',
-      exclusive: ['account', 'auto-approve', 'auto_approve', 'refresh'],
-    }),
-    production: flags.boolean({
```

Comment:
Sorry for the merge conflicts from:
https://github.com/architect-team/architect-cli/commit/73f5aa2036c9e543c3b544f54f2e259dd9e3a310